CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # PLCrashReporter Change Log
 
+## Version 1.11.1 (Under development)
+
+* **[Fix]** Disable "TREAT_WARNINGS_AS_ERRORS"
```

Comment:
It says nothing to customers. Please, remove this line if it's only about changing project internals or explain what scenario is actually fixed